src/main/histFileMod.F90

```diff
@@ -318,6 +330,78 @@ subroutine hist_printflds()
        call shr_sys_flush(iulog)
     end if
 
+    ! Print master field list in separate file if namelist variable
+    ! requests it
+    if (masterproc .and. hist_master_list_file) then
+       ! hardwired table column widths
+       width_col_1 = 4  ! column that shows the variable number, nf
+       width_col_2 = 39  ! variable name column
+       width_col_3 = 94  ! long description column
+       width_col_4 = 65  ! units column
```

Comment:
- [ ] [optional] should these be determined more dynamically, for example, based on the actual maximum length of variable names, etc.?

Reply:
@billsacks your suggestion makes sense on one hand. On the other, I was thinking of controlling the size of this table so that each row would fit on a computer screen. For long names that get cut off, users could still search for the complete description in the code.

I'm open to going with your suggestion if that's preferred.

Reply:
Unless... long names will end up wrapping in the available space. I didn't think of that earlier.

Reply:
Your rationale makes sense. Thanks for explaining.

Reply:
I have built the new .rst file's html version. Columns with long strings do wrap into the next line(s), so that's good news. The issue of fitting on the screen still applies when looking at the .rst file itself.

So I would still like final feedback on the subject of including the complete length of long_name. It's less of an issue with the other columns.

Reply:
To take a look and offer comments, you will find the relevant html files (master_list_file.html and customizing-the-clm-namelist.html) here:
`/glade/work/slevis/git/ctsm-docs/versions/master/html/users_guide/setting-up-and-running-a-case`

Reply:
I'll defer to others (@danicalombardozzi @katiedagon @wwieder @dlawrenncar @olyson etc.)

Reply:
I think this makes sense. It doesn't have a huge impact since it only cuts off long name descriptions for a few variables, and in all cases the primary relevant information is still included.